backend/src/apiserver/visualization/types/tfdv.py

```diff
@@ -12,14 +12,81 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import base64
 import tensorflow_data_validation as tfdv
+from IPython.display import display
+from IPython.display import HTML
+from tensorflow_metadata.proto.v0 import statistics_pb2
+from typing import Text
 
 # The following variables are provided through dependency injection. These
 # variables come from the specified input path and arguments provided by the
 # API post request.
 #
 # source
 
-train_stats = tfdv.generate_statistics_from_csv(data_location=source)
+# train_stats = tfdv.generate_statistics_from_csv(data_location=source)
+# tfdv.visualize_statistics(train_stats) 
 
-tfdv.visualize_statistics(train_stats)
+def get_statistics_html(
```

Comment:
after TFDV fix got release, here it can be reverted and may move to a new type e.x. tfdvstats.py